Sources/MapboxCoreNavigation/EHorizon/RoadGraphEdgeMetadata.swift

```diff
@@ -90,7 +93,8 @@ extension RoadGraph.Edge {
                     countryCode: String?,
                     regionCode: String?,
                     drivingSide: DrivingSide,
-                    directionality: Directionality) {
+                    directionality: Directionality,
+                    isUrban: Bool) {
```

Comment:
Adding this parameter would be a backwards-incompatible change, so give it a default value of `false`. We’ll just have to assume that the edge is non-urban if left unspecified. (We could make the `isUrban` property optional too, but at some point we’d have to translate it to `false` anyways when converting to the MapboxNavigationNative object.)

Reply:
Should we use a convenience init since providing a default value is still backwards-incompatible (according to the bot)? I think we've done similar changes before, and they caused issues later on.